From 680a5dd2a492078a0cc497502aab1d81cdba39ac Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Mon, 3 Sep 2012 11:10:01 +0200
Subject: [PATCH 1/2] Fix incorrect error on UpdateLocalMediaDescription

Remove the code that errors out a local description update if there is
*any* outstanding offer. A bit later in the same function a proper check
is done to see if there is a offer for the update contact, which is
correct.
---
 telepathy-glib/base-media-call-content.c |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/telepathy-glib/base-media-call-content.c b/telepathy-glib/base-media-call-content.c
index a0032f4..b7cf8e1 100644
--- a/telepathy-glib/base-media-call-content.c
+++ b/telepathy-glib/base-media-call-content.c
@@ -683,14 +683,6 @@ tp_base_media_call_content_update_local_media_description (
   GPtrArray *codecs;
   gpointer contact;
 
-  if (self->priv->current_offer != NULL)
-    {
-      GError error = { TP_ERROR, TP_ERROR_NOT_AVAILABLE,
-          "There is a media description offer around so "
-          "UpdateMediaDescription shouldn't be called." };
-      dbus_g_method_return_error (context, &error);
-      return;
-    }
 
   if (!g_hash_table_lookup_extended (properties,
           TP_PROP_CALL_CONTENT_MEDIA_DESCRIPTION_REMOTE_CONTACT,
-- 
1.7.10.4

